src/jobs/grype_image_scan.yml

```diff
@@ -0,0 +1,77 @@
+description: |
+  This job scans the Docker image for vulnerabilities using Grype.
+executor: machine
+environment:
+  MAIN_BRANCH_NAME: main
```

Comment:
we may have to parameterize this or provide input that works both for "main" and "master" named primary branches